doc/setup/install.rst

```diff
@@ -110,7 +110,7 @@ Linux
 Debian/Ubuntu
 '''''''''''''
 
-Odoo 12.0 'deb' package currently supports `Debian Stretch`_, `Ubuntu 18.04`_ or above.
+Odoo 13.0 'deb' package currently supports `Debian Stretch`_, `Ubuntu 18.04`_ or above.
```

Comment:
To check with @d-fence but I think we support Debian Buster now

Reply:
I confirm. It's Debian Buster now, as it's the new Debian stable

Reply:
Change to Buster done!